### PR TITLE
DataGrid Export - Support column.alignment property

### PR DESCRIPTION
**Pull request by KuznetsovVN**

### CODE REVIEW

File: js/exporter/jspdf/v3/rows_generator.js [hunk 1]
```diff
@@ -62,16 +62,16 @@ function generateRowsInfo(dataProvider, dataGrid, headerBackgroundColor) {
     return result;
 }
 
-function generateRowCells({ dataProvider, rowIndex, wordWrapEnabled, colCount, rowType, backgroundColor }) {
+function generateRowCells({ dataProvider, rowIndex, wordWrapEnabled, columns, colCount, rowType, backgroundColor }) {
```
Comment (novsstation) [inline js/exporter/jspdf/v3/rows_generator.js]: if we pass whole columns, then colCount argument is redundant.

**Review by novsstation — DISMISSED** [review]

File: js/exporter/jspdf/v3/rows_generator.js (hunk 1 above)
Comment (KuznetsovVN) [reply js/exporter/jspdf/v3/rows_generator.js]: Ok

File: testing/tests/DevExpress.exporter/jspdfParts/v3/jspdf_v3.dataGrid.horizontalAlign.tests.js [hunk 2]
```diff
@@ -792,6 +792,1195 @@ const JSPdfHorizontalAlignTests = {
...
+                ];
+
+                exportDataGrid(doc, dataGrid, { topLeft: { x: 10, y: 15 }, columnWidths: [ 200 ] }).then(() => {
+                    // doc.save(assert.test.testName + '.pdf');
+                    assert.deepEqual(doc.__log, expectedLog);
+                    done();
+                });
+            });
+
+            QUnit.test('1 col - 3 row. col.dataType: number, col.alignment: left, pdfCell.alignment: left, Cell width = 200px ', function(assert) {
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/jspdfParts/v3/jspdf_v3.dataGrid.horizontalAlign.tests.js]: Many tests for [type, col.alignment, pdfCell.alignment] combinations looks redundant
imho it's enough {string, number} x {col.default , col.center} x {pdfCell. right}

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.exporter/jspdfParts/v3/jspdf_v3.dataGrid.horizontalAlign.tests.js (hunk 2 above)
Comment (KuznetsovVN) [reply testing/tests/DevExpress.exporter/jspdfParts/v3/jspdf_v3.dataGrid.horizontalAlign.tests.js]: Ok, I reduced the combination of tests. At the moment there are the main scenarios affecting the redefinition of alignment value.

**Review by novsstation — APPROVED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**